spyder/config/main.py

```diff
@@ -420,7 +420,7 @@
             ('color_schemes',
              {
               'names': ['emacs', 'idle', 'monokai', 'pydev', 'scintilla',
-                        'spyder', 'spyder/dark', 'zenburn'],
+                        'spyder', 'spyder/dark', 'zenburn', 'solarized', 'solarized/dark'],

```

Comment:
Sorry for asking you to change this again, but I looked at several editors and IDEs, and use "Solarized Light" as the name for the light version of Solarized. So please all variables to `solarized/light`